### PR TITLE
Consolidate k8s setup docs

### PR DESCRIPTION
**Pull request by duboisph**
Consolidate k8s setup docs to our documentation as per https://skyscrapers.slack.com/archives/C04P0T47V/p1538061713000100.

Related PR: https://github.com/skyscrapers/internal-documentation/pull/61

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,132 +62,4 @@ module "kops-aws" {
...
-#### Helm packages
-
-The same applies to the deployed Helm packages. If an update needs to be made, just taint the `null_resource.helm_values_file` resource in the base module and rerun `terraform apply`. Then with the new `helm-values.yaml` file you'll be able to upgrade all the helm packages by doing:
-
-```
-helm upgrade <release_name> skyscrapers/<helm_chart_name> --values helm-values.yaml
-```
-
-Please refer to our [Helm charts documentation](https://github.com/skyscrapers/charts/blob/master/README.md) for an overview of the base packages we deploy.
+[Please refer to our documentation on how to setup a cluster.](https://github.com/skyscrapers/tree/master/services/kubernetes/setup.md)
```
Comment (iuriaranda) [inline README.md]: The URL is wrong, the repo name is missing.

**Comment by duboisph** [comment]
@iuriaranda The documentation is very specific in how we set things up. Also, now it is public, but since we are moving the modules to a (private) stack it will no longer be relevant either way.

**Comment by duboisph** [comment]
@iuriaranda Then instead of removing the doc altogether or making this repo private, I propose to remove all the base/bootstrap docs so it's cleaned up to only the relevant parts for this module. I will also add a note that for our own use we should refer to our documentation repo instead.

That way the relevant documentation still is accessible as long as the repo exists.

**Comment by iuriaranda** [comment]
ok for me 👍

**Pull request merged**